shaders.go

```diff
@@ -232,6 +242,10 @@ func (s *basicShader) generateBufferContent(ren *RenderComponent, buffer []float
 	h := ren.Drawable.Height()
 
 	colorR, colorG, colorB, colorA := ren.Color.RGBA()
+	colorR >>= 8
+	colorG >>= 8
+	colorB >>= 8
+	colorA >>= 8
```

Comment:
Because `ren.Color.RGBA()` returns in the range of `0` to `0xffff` (instead of `0xff`), they need to be made smaller in order to work. This is being done in other parts of the code as well. 

Note this will result in a range from `0` to `255` (`0xff`). The `SetBackground` expected range from `0` to `1`, which is why we divided by `0xffff` in that scenario.